OracleManagementAgent/dockerfiles/latest/container-scripts/initialize.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# Copyright (c) 2023 Oracle and/or its affiliates.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+set -eu  # Exit on error
+set -o pipefail  # Fail a pipe if any sub-command fails.
+
+###########################################################
+# Global constants
+STAGING_HOME=/opt/oracle-mgmtagent-staging
+BOOTSTRAP_HOME=/opt/oracle/bootstrap
```

Comment:
Why are you using `oracle-mgmtagent-staging` and then just `oracle`? It would be better if these were more consistent. Then they could be used on the host as well with little chance of already existing.

Reply:
if /opt/oracle already exists inside the container then a external shared mount will fail at that location. oracle-mgmtagent-staging is files copied into the container at build time and /opt/oracle is the shared mount that is created at runtime. They have to be separate locations.